### DISCUSSION TITLE
7/05-24

### DISCUSSION THREAD

**Discussion by Pamela198713** [enbalde-ispc/enbalde-ispc#107] [Daily Sprint]
Qué hice ayer
Qué hago hoy
Impedimentos

**Comment by Pamela198713** [comment]
tuve en una clase. A la salida me reuni con el equiipo para avanzar en Emprendedurismo y algo de Tecnologiaa
Hoy tengo una clase, luego me pongo con Lu y Yu hacer lo de emprendedurismo.
Los tiempos y la organizacion en la casa.

**Comment by rpgrca** [comment]
Ayer nos juntamos para empezar con lo de Emprendedurismo y Tecnología y Desarrollo. Hoy tengo clase de Emprendedurismo como para tener información de lo último que hay que hacer. Además apenas empecé con la lógica de la registración en la app, espero poder terminarla hoy para poder empezar con el carrito después.

**Comment by fmdcba** [comment]
Ayer participé de la reunión grupal, hoy estuve leyendo sobre lean startup y trabajando en el proyecto emprendedurismo y corrigiendolo en base a lo que se charló ayer en grupo.
